File: Examples/CSharp/Articles/SettingPivotTableOption.cs
using System.IO;

using Aspose.Cells;
using Aspose.Cells.Pivot;

namespace Aspose.Cells.Examples.CSharp.Articles
{
    public class SettingPivotTableOption
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);

            Workbook wb = new Workbook(dataDir + "input.xlsx");

            // PivotTable pt = wb.Worksheets[0].PivotTables[0];

            // Indicating if or not display the empty cell value
            // Pt.DisplayNullString = true;

            // Indicating the null string
            // Pt.NullString = "null";

           // pt.CalculateData();

           // pt.RefreshDataOnOpeningFile = false;

           // wb.Save(dataDir+ "output.out.xlsx");
            
        }
    }
}

File: Examples/VisualBasic/Charts/SettingPivotTableOption.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Cells
Imports Aspose.Cells.Pivot

Namespace Articles
    Public Class SettingPivotTableOption
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            Dim wb As New Workbook(dataDir & "input.xlsx")

            Dim pt As PivotTable = wb.Worksheets(0).PivotTables(0)

            ' Indicating if or not display the empty cell value
            pt.DisplayNullString = True

            ' Indicating the null string
            pt.NullString = "null"

            pt.CalculateData()

            pt.RefreshDataOnOpeningFile = False

            wb.Save(dataDir & "output.xlsx")
            ' ExEnd:1

        End Sub
    End Class
End Namespace